radar/languages-and-frameworks/fp-ts.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "fp-ts"
+ring: trial
+quadrant: "languages-and-frameworks'
+tags: [functional]
+---
+
+The [fp-ts](https://gcanti.github.io/fp-ts/) is a powerful library that brings functional programming concepts to TypeScript development. It provides a set of functional programming abstractions and utilities for building robust and type-safe applications. Leveraging algebraic data types, type classes, and immutable data structures, fp-ts enables developers to write functional, expressive, and statically-typed code in TypeScript.
```

Comment:
I'd add a short paragraph (Use cases) that states why we use fp-ts (error handling, composition of synchronous and asynchronous tasks).